ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml

```diff
@@ -211,22 +211,19 @@ StatusModal {
             spacing: Style.current.padding
             Layout.fillWidth: true
             Layout.fillHeight: true
-            model: popup.channel? popup.channel.members : []
+            model: popup.store.currentChatContentModule().usersModule.model//popup.channel? popup.channel.members : []
             delegate: StatusListItem {
                 id: contactRow
 
-                property string nickname: Utils.getContactDetailsAsJson(model.publicKey).localNickname
+                //property string nickname: Utils.getContactDetailsAsJson(model.publicKey).localNickname
 
-                title: !model.userName.endsWith(".eth") && !!contactRow.nickname ?
-                           contactRow.nickname : Utils.removeStatusEns(model.userName)
-                //% "(You)"
-                titleAsideText: model.publicKey === userProfile.pubKey ? qsTrId("-you-") : ""
+                title: model.name
                 //% "Admin"
                 statusListItemTitle.font.pixelSize: 17
                 statusListItemTitleAside.font.pixelSize: 17
                 label: model.isAdmin ? qsTrId("group-chat-admin"): ""
-                image.source: Global.getProfileImage(model.publicKey) || model.identicon
-                image.isIdenticon: model.identicon
+                image.source: model.icon//Global.getProfileImage(model.publicKey) || model.identicon
```

Comment:
@borismelnik delete this part since we don't need it any more `//Global.getProfileImage(model.publicKey) || model.identicon`